typings/model.d.ts

```diff
@@ -133,6 +133,31 @@ declare interface Foul {
   [key: string]: any
 }
 
+interface Franchise {
+  apiCode: string
+  league: Schema.Types.ObjectId
+  ownership: string[]
+  affliation: string
+  team: Schema.Types.ObjectId
+  teamCity: string
+  teamName: string
+  teamAbbreviation: string
+  founded: string
+  history: string
+  arena: string
+  years: number
+  games: number
+  wins: number
+  losses: number
+  winPercentage: number
+  playoffAppearances: number
+  divisionTitles: number
+  conferenceTitles: number
+  championships: number
+  teamHistory: Schema.Types.ObjectId[]
```

Comment:
Change this to an Object array. instead of ObjectId, we'll reference the main if needed.

Reply:
will do..